app/komponenter/tekstBlokk/tekstBlokk.tsx

```diff
@@ -8,9 +8,14 @@ import { flettefeltTilTekst } from '~/utils/fletteTilTekst';
 interface Props {
   tekstblokk: SanityDokument | undefined;
   typografi?: TypografiTyper;
+  flettefeltInnhold?: string;
```

Comment:
Hva skjer om vi har to forskjellige flettefelter i samme tekst nå?

Reply:
Det fungerer jo dårlig ja, så det er endret her [5ccd956](https://github.com/bekk/nav-familie-endringsmelding/pull/20/commits/5ccd9562de7b21b79efec9056bffd01580ee1934)